### ISSUE TITLE
remove "U" button 

### ISSUE THREAD

**Issue by janzandr** [EnMAP-Box/enmap-box#156]
_Requested by Sebastian van der Linden._

Remove the "U" button for floated views...
![image](https://user-images.githubusercontent.com/45030715/194731241-aa7f014f-ec24-48b9-b4bc-22e817c648ff.png)

... because it's functionality is already given by the "X" button:
![image](https://user-images.githubusercontent.com/45030715/194731270-a50e3d02-22ec-4eed-9846-b5fd3c8df754.png)

**Comment by jakimowb** [comment]
I think there is a missunderstanding. "Float" dock means to create a new DockArea and add the dock to. After that, other docks may get moved to the new DockArea, e.g. to show them on a 2nd screen, while the main EnMAP-Box window remains on the 1st screen.

![grafik](https://github.com/EnMAP-Box/enmap-box/assets/1404870/916e0370-cd3f-4ef9-a769-ba767ad9e8d2)

Closing the floated DockArea II (big X) moves all if its docks (dock c+d) back to DockArea I.
The "U" button allows to move a single dock only back to DockArea I (and "X" to close it for ever).

Up to this point I think we should use it as it is. What do you think @janzandr ?

**Comment by janzandr** [comment]
Yes, technically it is correct as you explained it. But I can also see, where Sebastians and my confusion arises from: clicking the "Float" button, doesn't actually make a view floating, but just docks the view into a new DockArea :-)

So maybe tooltip **"Float window"** should be replaced by **"Move view to new dock area"**.
